package.json

```diff
@@ -22,14 +22,18 @@
     "chroma-js"
   ],
   "sideEffects": false,
-  "main": "./dist/index.js",
-  "unpkg": "./dist/index.umd.js",
-  "module": "./dist/index.module.js",
-  "source": "./src/index.ts",
+  "main": "./dist/index.main.umd.js",
+  "module": "./dist/index.module.es.js",
   "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.exports.import.es.mjs",
+      "require": "./dist/index.exports.require.cjs.js"
+    },
```

Comment:
```suggestion
    ".": [
      {
        "import": "./dist/index.exports.import.es.mjs",
        "require": "./dist/index.exports.require.cjs.js",
        "default": "./dist/index.exports.require.cjs.js"
      },
      "./dist/index.exports.require.cjs.js"
    ],
```
otherwise it'll break in a few node 13 versions. you could also just omit "require" entirely, but you always need "default".

Reply:
ah nice. i didn't know it supported an array. i'll give this a try this week

---

>  you could also just omit "require" entirely, but you always need "default".

I'm looking at `@rollup/plugin-node-resolve` and it says:

> By default, this plugin looks for the `['default', 'module', 'import']` conditions when resolving imports.
>
> [[source]](https://github.com/rollup/plugins/tree/9874740ca685b1600e4319365c21e8c1131e9d95/packages/node-resolve#exportconditions)

Do you know if the order matters there? I intentionally left out `default` there because I wanted the rollup plugin to resolve `module` before default